list.js

```diff
@@ -53,31 +53,31 @@
     };
 
     var clearSearch = function() {
-        $('#calendarList').empty();
-        $('#calendarList').hide();
+        jQuery('#calendarList').empty();
+        jQuery('#calendarList').hide();
     };
 
     /**
      * Generate an element containing all the events that belong on
      * the given page number.
      */
     var renderEvents = function(eArray, pageNum) {
-        var $container = jQuery('<div class="ctl-events" />');
+        var jQuerycontainer = jQuery('<div class="ctl-events" />');
```

Comment:
no need to update these variable names - `$` in a variable name is treated just like `jQuery`, and has no special meaning.